cmd/node/config/prefs.toml

```diff
@@ -45,14 +45,10 @@
    # -------------------------------
    # Un-comment and update the following section in order to enable config values overloading
    # -------------------------------
-   # OverridableConfigTomlValues = [
-      # { File = "config.toml", Path = "StoragePruning.NumEpochsToKeep", Value = 4 },
-      # { File = "config.toml", Path = "MiniBlocksStorage.Cache.Name", Value = "MiniBlocksStorage" },
-      # { File = "external.toml", Path = "ElasticSearchConnector.Enabled", Value = true },
-      # { File = "external.toml", Path = "HostDriversConfig", Value = [
-      #    { Enabled = false, URL = "127.0.0.1:22111" },
-      # ] },
-   # ]
+    OverridableConfigTomlValues = [
```

Comment:
We should change the sovereign's node config prefs.toml, not the node's config.
Also, probably scripts for local testnet have to be updated as well

Reply:
added prefs file for sovereign, also changed scripts